src/Features/CSharpTest/SimplifyTypeNames/SimplifyTypeNamesTests.cs

```diff
@@ -7278,8 +7274,48 @@ await TestMissingInRegularAndScriptAsync(
             """, new TestParameters(options: featureOptions));
     }
 
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/74973")]
+    public async Task TestNint1_WithNumericIntPtr_CSharp11_NoRuntimeSupport()
+    {
+        await TestInRegularAndScript1Async("""
+            <Workspace>
```

Comment:
><Workspace> [](http://example.com/codeflow?start=12&length=11)

For my benefit, why does this use the markup/workspace string format rather than just specifying the string as most of the tests do in this file?

Reply:
because it needs to specify the set of references/runtime the project is configured against.  For the legacy test framework (the one that is not using the `VerifyCS` pattern) this can only be done with this `<Workspace>` format.  For the modern test framework, you can do `ReferenceAssemblies = ReferencesAssemblies.Net.Net80` and stuff like that.